store/classes.patterns.js

```diff
@@ -94,12 +95,98 @@ const classPatterns = {
+  [`^inset-block-start${digitPattern}$`]: ".inset-block-start-$1 { inset-block-start: $1px; }",
+  [`^inset-inline-${digitPattern}$`]: ".inset-inline-$1 { inset-inline: $1px; }",
+  [`^inset-inline-${digitPattern}$`]: ".inset-inline-$1-$2 { inset-inline: $1px $2px; }",
+  [`^inset-inline-end${digitPattern}$`]: ".inset-inline-end-$1 { inset-inline-end: $1px; }",
+  [`^inset-inline-start${digitPattern}$`]: ".inset-inline-start-$1 { inset-inline-start: $1px; }",
+
+  [`^letter-spacing-${digitPattern}$`]: ".letter-spacing-$1 { letter-spacing: $1px; }",
+  [`^max-block-size-${digitPattern}$`]: ".max-block-size-$1 { max-block-size: $1px; }",
+  [`^max-inline-size-${digitPattern}$`]: ".max-inline-size-$1 { max-inline-size: $1px; }",
+  [`^min-block-size-${digitPattern}$`]: ".min-block-size-$1 { min-block-size: $1px; }",
+  [`^min-inline-size-${digitPattern}$`]: ".min-inline-size-$1 { min-inline-size: $1px; }",
+
+  [`^object-position-${digitPattern}-${digitPattern}$`]: ".object-position-$1-$2 { object-position: $1px $2px; }",
+  [`^object-position-${digitPattern}-${digitPattern}(p)$`]: ".object-position-$1-$2p { object-position: $1px $2%; }",
+  [`^object-position-${digitPattern}(p)-${digitPattern}$`]: ".object-position-$1p-$2 { object-position: $1% $2px; }",
+  [`^object-position-${digitPattern}(p)-${digitPattern}(p)$`]: ".object-position-$1p-$2p { object-position: $1% $2%; }",
+  [`^offset-${digitPattern}$`]: ".offset-$1 { offset-distance: $1px; }",
+  [`^offset-${digitPattern}(p)$`]: ".offset-$1p { offset-distance: $1%; }",
+
+  [`^order-${digitPattern}$`]: ".order-$1 { order: $1; }",
+
+  [`^orphans-${digitPattern}$`]: ".orphans-$1 { orphans: $1; }",
+
+  [`^perspective-${digitPattern}$`]: ".perspective-$1 { perspective: $1px; }",
+  [`^perspective-${digitPattern}(p)$`]: ".perspective-$1p { perspective: $1%; }",
+
+  [`^row-gap-${digitPattern}$`]: ".row-gap-$1 { row-gap: $1px; }",
+
+  [`^scale-${digitPattern}$`]: ".scale-$1 { scale: $1; }",
+  [`^scale-${digitPattern}-${digitPattern}$`]: ".scale-$1-$2 { scale: $1 $2; }",
+  [`^scale-${digitPattern}-${digitPattern}-${digitPattern}$`]: ".scale-$1-$2-$3 { scale: $1 $2 $3; }",
+
+  [`^scroll-m-${digitPattern}$`]: ".scroll-m-$1 { scroll-margin: $1px; }",
+  [`^scroll-m-${digitPattern}-${digitPattern}$`]: ".scroll-m-$1-$2 { scroll-margin: $1px $2px; }",
+  [`^scroll-m-${digitPattern}-${digitPattern}-${digitPattern}$`]: ".scroll-m-$1-$2-$3 { scroll-margin: $1px $2px $3px; }",
+  [`^scroll-m-${digitPattern}-${digitPattern}-${digitPattern}-${digitPattern}$`]: ".scroll-m-$1-$2-$3-$4 { scroll-margin: $1px $2px $3px $4px; }",
+  [`^scroll-mt-${digitPattern}$`]: ".scroll-mt-$1 { scroll-margin: $1px; }",
+  [`^scroll-mb-${digitPattern}$`]: ".scroll-mb-$1 { scroll-margin: $1px; }",
+  [`^scroll-ml-${digitPattern}$`]: ".scroll-ml-$1 { scroll-margin: $1px; }",
+  [`^scroll-mr-${digitPattern}$`]: ".scroll-mr-$1 { scroll-margin: $1px; }",
```

Comment:
Looks like the rules in these scroll-margin and padding are all the same. Kindly review.